core/src/main/java/org/ethereum/beacon/core/types/Gwei.java

```diff
@@ -33,6 +33,23 @@ public Gwei minus(Gwei subtrahend) {
     return new Gwei(super.minus(subtrahend));
   }
 
+  public Gwei minusSat(Gwei subtrahend) {
```

Comment:
what does it mean?

Reply:
It's a reference to [saturation arithmetic](https://en.wikipedia.org/wiki/Saturation_arithmetic). Maybe you can come with a better naming to this method?

Reply:
Just add javadoc comment to `plusSat` and `minSat` like in `UInt64`